bentoml/yatai/yatai_service_impl.py

```diff
@@ -578,6 +592,114 @@ def ContainerizeBento(self, request, context=None):
                     logger.error(f"RPC ERROR ContainerizeBento: {e}")
                     return ContainerizeBentoResponse(status=Status.INTERNAL(e))
 
-            # pylint: enable=unused-argument
+        def UploadBento(self, request_iterator, context=None):
+            if not is_file_system_repo(self.repo):
+                logger.error(
+                    "Yatai only supports file system repository for uploading "
```

Comment:
nit: "UploadBento RPC only works with File System based repository, for other types of repositories(s3, gcs, minio), use pre-signed URL for download."